src/Controls/src/Core/HandlerImpl/Button/Button.Impl.cs

```diff
@@ -4,21 +4,21 @@
 
 namespace Microsoft.Maui.Controls
 {
-	public partial class Button : IButton
+	public partial class Button : IButton, IText
```

Comment:
Removed `IText` from `IButton` because `Button` might not be a `Text` based button.